doc/sphinx/source/changelog.rst

```diff
@@ -3,6 +3,112 @@
 Changelog
 =========
 
+.. _changelog-v2-11-0:
+
+v2.11.0
+-------
+Highlights
+
+- New CMORizers have been added:
+
+  - Updating ESACCI-WATERVAPOUR cmorizer (:pull:`3282`) by :user:`malininae`
+  - CMORiser for ANU Climate 2.0 Australian data (:pull:`3511`) by :user:`flicj191`
+
+- NEW TREND: First time release manager shout-outs!
+
+  - This is the first ESMValTool release managed by the Met Office! We want to
+    shout this out - and for all future first time release managers to
+    shout-out - to celebrate the growing, thriving ESMValTool community.
+
+This release includes
+
+Bug fixes
+~~~~~~~~~
+
+-  Update OceanSODA-ETHZ CMORizer with new source file (:pull:`3535`) by :user:`TomasTorsvik`
+-  Fix for ``recipe_seaice_drift.yml``: fix CRS transformer for "North Pole Stereographic" (:pull:`3531`) by :user:`flicj191`
+-  Recipe_ocean_quadmap: Update ATSR to match ESGF name (:pull:`3443`) by :user:`rbeucher`
+-  Fixed attribute handling in austral_jet/main.ncl for iris>=3.8 (:pull:`3603`) by :user:`schlunma`
+-  Update the name of the remapcon2 operator in R recipes (:pull:`3611`) by :user:`ehogan`
+-  Fix recipe_bock20jgr_fig_8-10.yml (:pull:`3665`) by :user:`LisaBock`
+
+Documentation
+~~~~~~~~~~~~~
+
+-  Improve release tools and documentation (:pull:`3462`) by :user:`bouweandela`
+-  Fix a typo in the references file (:pull:`3499`) by :user:`bouweandela`
+-  Fix recipe path in ``recipe_perfmetrics.rst`` (:pull:`3532`) by :user:`TomasTorsvik`
+-  Improved description of model evaluation recipes (:pull:`3541`) by :user:`schlunma`
+-  Remove double word in cmorizer documentation (:pull:`3553`) by :user:`bettina-gier`
+-  Fix Codacy badge (:pull:`3558`) by :user:`bouweandela`
+-  Update the release schedule for v2.11.0 (:pull:`3573`) by :user:`ehogan`
+-  Improve the formatting of the recipe documentation template (:pull:`3652`) by :user:`mo-gill`
+-  Add introduction material on the main documentation page (:pull:`3628`) by :user:`bouweandela`
+-  Avoid warning in documentation build (:pull:`3675`) by :user:`bouweandela`
+
+Diagnostics
+~~~~~~~~~~~
+
+-  ``monitor/multi_dataset.py`` improvements: allow data w/o ``timerange`` and improve text formatting (:pull:`3528`) by :user:`schlunma`
+-  Allow datasets without ``project`` in multi_datasets.py (:pull:`3552`) by :user:`schlunma`
+-  Fixed attribute handling in emergent constraint diagnostic for iris>=3.8 (:pull:`3605`) by :user:`schlunma`
+-  Use ``iris.FUTURE.save_split_attrs = True`` to remove iris warning in many diagnostics (:pull:`3651`) by :user:`schlunma`
+-  Avoid concatenation error in recipe_pcrglobwb.yml (:pull:`3645`) by :user:`bouweandela`
```

Comment:
These are probably also bugfixes

Reply:
Or maybe not, let's leave them here. They're upgrades so the older code works with new dependencies, I don't think we have a good category for that.